script.js

```diff
@@ -7,37 +27,76 @@ function createProductImageElement(imageSource) {
   return img;
 }
 
-function createCustomElement(element, className, innerText) {
-  const e = document.createElement(element);
-  e.className = className;
-  e.innerText = innerText;
-  return e;
+function cartItemClickListener(event) {
+  event.target.remove();
+  storedCart();
+}
+
+function createCartItemElement({ sku, name, salePrice }) {
+  const li = document.createElement('li');
+  li.className = 'cart__item';
+  li.innerText = `SKU: ${sku} | NAME: ${name} | PRICE: $${salePrice}`;
+  li.addEventListener('click', cartItemClickListener);
+  return li;
 }
 
+const cartAdd = async (idSku) => {
+  loading();
+  const cartList = document.getElementsByClassName('cart__items')[0];
+  const cartItem = await fetchAPI(`https://api.mercadolibre.com/items/${idSku}`);
+  cartList.appendChild(createCartItemElement({ sku: cartItem.id,
+    name: cartItem.title,
+    salePrice: cartItem.price }));
+  storedCart();
+  notLoading();
+};
+
 function createProductItemElement({ sku, name, image }) {
   const section = document.createElement('section');
   section.className = 'item';
 
   section.appendChild(createCustomElement('span', 'item__sku', sku));
   section.appendChild(createCustomElement('span', 'item__title', name));
   section.appendChild(createProductImageElement(image));
-  section.appendChild(createCustomElement('button', 'item__add', 'Adicionar ao carrinho!'));
-
+  const addBtn = section.appendChild(createCustomElement('button', 'item__add', 'Adicionar ao carrinho!'));
+  addBtn.addEventListener('click', () => cartAdd(sku));
   return section;
 }
 
 function getSkuFromProductItem(item) {
   return item.querySelector('span.item__sku').innerText;
 }
 
-function cartItemClickListener(event) {
-  // coloque seu código aqui
-}
+const clrBtn = document.getElementsByClassName('empty-cart')[0];
 
-function createCartItemElement({ sku, name, salePrice }) {
-  const li = document.createElement('li');
-  li.className = 'cart__item';
-  li.innerText = `SKU: ${sku} | NAME: ${name} | PRICE: $${salePrice}`;
-  li.addEventListener('click', cartItemClickListener);
-  return li;
-}
+clrBtn.addEventListener('click', () => {
+  const list = document.getElementsByClassName('cart__items')[0];
+  list.innerHTML = '';
+  storedCart();
+});
+
+const loadCart = () => {
+  document.getElementsByClassName('cart__items')[0].innerHTML = localStorage.getItem('cart-items');
+};
+
+window.onload = function onload() {
+  loadCart();
+  document.getElementsByClassName('cart__items')[0].childNodes.forEach(e =>
+    e.addEventListener('click', cartItemClickListener));
+
+  setTimeout(() => {
+    document.querySelector('.loading').remove();
```

Comment:
Outra forma de remover o loading aqui seria reaproveitando a sua função notLoading.
```suggestion
    notLoading();
```